<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSerialPortTestClass</class>
 <widget class="QMainWindow" name="QSerialPortTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSerialPortTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portComboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="portStatusLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status port : Fermé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir le port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Données reçus :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="receivedDataText">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>89</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QSerialPortTest.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QSerialPortTestClass</receiver>
   <slot>onOpenPortButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QSerialPortTestClass</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenPortButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
 </slots>
</ui>
